src/views/ProductView.vue

```diff
@@ -578,4 +622,247 @@ function goToWishlists() {
+ * @returns The number of product items in fulfillment or undefined if no data is available.
+ */
+const numberOfProductItemsInFullfillment = computed(() => {
+    if (inventoryStatusOfProductItems.value !== null) {
+        return inventoryStatusOfProductItems.value.filter(
+            (productItem) =>
+                productItem.inventoryStatus === ProductItemStatus.InFulfillment
+        ).length
+    }
+
+    return undefined
+})
+
+/**
+ * Computed property for the number of shipped product items.
+ * @returns The number of shipped product items or undefined if no data is available.
+ */
+const numberOfShippedProductItems = computed(() => {
+    if (inventoryStatusOfProductItems.value !== null) {
+        return inventoryStatusOfProductItems.value.filter(
+            (productItem) =>
+                productItem.inventoryStatus === ProductItemStatus.Shipped
+        ).length
+    }
+
+    return undefined
+})
+
+/**
+ * Computed property for the number of delivered product items.
+ * @returns The number of delivered product items or undefined if no data is available.
+ */
+const numberOfDeliveredProductItems = computed(() => {
+    if (inventoryStatusOfProductItems.value !== null) {
+        return inventoryStatusOfProductItems.value.filter(
+            (productItem) =>
+                productItem.inventoryStatus === ProductItemStatus.Delivered
+        ).length
+    }
+
+    return undefined
+})
+
+/**
+ * Computed property for the number of returned product items.
+ * @returns The number of returned product items or undefined if no data is available.
+ */
+const numberOfReturnedProductItems = computed(() => {
+    if (inventoryStatusOfProductItems.value !== null) {
+        return inventoryStatusOfProductItems.value.filter(
+            (productItem) =>
+                productItem.inventoryStatus === ProductItemStatus.Returned
+        ).length
+    }
+
+    return undefined
+})
+
+/**
+ * Computed property for the number of lost product items.
+ * @returns The number of lost product items or undefined if no data is available.
+ */
+const numberOfLostProductItems = computed(() => {
+    if (inventoryStatusOfProductItems.value !== null) {
+        return inventoryStatusOfProductItems.value.filter(
+            (productItem) =>
+                productItem.inventoryStatus === ProductItemStatus.Lost
+        ).length
+    }
+
+    return undefined
+})
+
+/**
+ * The maximum number of product items a buyer can order.
+ *
+ * This computed ref calculates the maximum number of product items a buyer can order based on the
+ * number of items in stock and the number of reserved items.
+ * The available quantity is the number of items in stock minus the number of reserved items.
+ * If the available quantity is less than 10, it returns that value; otherwise,
+ * it returns the default maximum value of 10.
+ */
+const maximumNumberOfProductItemsABuyerCanOrder = computed(() => {
+    const defaultValue = 10
```

Comment:
constants somewhere in the code are generally not liked by Becker and co
maybe move to the top of the file as a global constant, or even introduce some constants / configuration file

Reply:
I moved it to the top of the file for now: b0e0e45dbda8c3316007e96ee48b807848fd4810